aztk/spark/client.py

```diff
@@ -146,15 +147,16 @@ def get_application_status(self, cluster_id: str, app_name: str):
         except batch_error.BatchErrorException as e:
             raise error.AztkError(helpers.format_batch_exception(e))
 
-    def cluster_run(self, cluster_id: str, command: str):
+    def cluster_run(self, cluster_id: str, command: str, host=False):
         try:
-            return self.__cluster_run(cluster_id, 'spark', command)
+            return self.__cluster_run(cluster_id, command, container_name='spark' if not host else None)
         except batch_error.BatchErrorException as e:
             raise error.AztkError(helpers.format_batch_exception(e))
 
-    def cluster_copy(self, cluster_id: str, source_path: str, destination_path: str):
+    def cluster_copy(self, cluster_id: str, source_path: str, destination_path: str, host=False, get=False):
```

Comment:
what does `get` do?

Reply:
get means retrieve files from the nodes. previously cluster copy was limited to copying a local file to all nodes on the cluster. Now it can work both ways.

Reply:
what would you suggest? is it just that the name of the parameter confusing?

Reply:
Not sure if having two methods would be more clear?
```python
def copy_to_cluster(...):
    ...

def copy_from_cluster(...):
    ...
```

Thoguhts?

Reply:
A change like that would be breaking for any script using `cluster_copy()` today, unless we have `cluster_copy()` and `copy_from_cluster()`. I feel like that is not the best naming.

In general, I think we should consider an entire SDK rewrite to align cluster and job function names (didn't do a particularly good job naming them). It would also be nice to split the client so it has a `cluster` and a `job` submodule. So you would do `client.cluster.get_log()` or `client.job.get_log()`.

Reply:
yeah I'm down to rename stuff. We can either keep the old stuff  to call the new function and mark as depractated or just remove them as we are technically only releasing it next version

Reply:
i think deprecating is probably better with something like that https://stackoverflow.com/questions/2536307/how-do-i-deprecate-python-functions

Reply:
Depreciating is fine, I think so long as we have a set time frame (maybe 1 or 2 versions) where we actually remove the code.